### ISSUE TITLE
Please use $HOME in SKILLS_FOLDER 

### ISSUE THREAD

**Issue by kuboosoft** [MycroftAI/mycroft-skills#119]
Why not use our home?

1) Fastly updates
2) Avoid privileges (/opt is bad)
3) Avoids security problems
4) Easy migration

`SKILLS_FOLDER="$HOME/.local/share/mycroft/skills"`

**Comment by penrods** [comment]
$HOME is user dependent, which doesn't work well.  Attempting to use $HOME was the first attempt, and it caused all sorts of confusion.  On the Mark 1/Raspberry Pi package install, the system runs as a MYCROFT user, so skills would have to be under there.  But then if you switched to a developer install that would be under the Pi user and none of the skills are found.  Similar issues on a desktop, plus I believe having consistent behavior for Mycroft across users (perhaps with custom skills loading in addition to the standards) makes more sense.

In short, since we made this change the number of confused people with regards to skills has plummeted.

**Comment by kjcole** [comment]
I don't have a horse in this race (yet). So no strong feelings one way or the other on where stuff should be located. 

However, I'm never shy about exposing my ignorance. ;-) Is it possible to improve the situation and somewhat satisfy both camps with some carefully crafted /etc/skel/* magic that creates one or more symlinks, plus an entry in /etc/groups to deal with permission issues?

**Comment by el-tocino** [comment]
Anything is possible if someone puts the effort into making it part of mycroft-core.  Closing this one for now, though, as it's not a skill per se, but more of a core issue.
